api/proto/info/info.proto

```diff
@@ -673,3 +676,10 @@ message ZInfoMsg {
   }
   google.protobuf.Timestamp atTimeStamp = 6;
 }
+
+// Information about running hypervisor and hardware capabilities
+message Capabilities {
+  string hypervisor = 1;
+  bool vtx = 2;
+  bool vtd = 3;
```

Comment:
What are you planning to put into these fields on ARM (and other architectures?) These ones are x86 specific.

Reply:
You mean, I should rephrase those lines?

Reply:
I mean, currently I have absolutely no idea what they mean ;-) and more importantly what is the semantics of them being true/false OR what strings should I expect in the string field.